release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -3358,4 +3358,23 @@ $ oc adm release info 4.12.7 --pullspecs
 [id="ocp-4-12-7-updating"]
 ==== Updating
 
+To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-12-8"]
+=== RHBA-2023:1269 - {product-title} 4.12.8 bug fix and security update
+
+Issued: 2023-03-20
+
+{product-title} release 4.12.8 which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:1269[RHBA-2023:1269] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:1268[RHSA-2023:1268] advisory.
```

Comment:
@JoeAldinger I suggest a comma after the version number to separate the independent clause "which includes security updates". The Release Note two before this one does so.

Reply:
I actually missed the comma, thanks @GroceryBoyJr .